profile/README.md

```diff
@@ -10,3 +10,45 @@
 🍿 Fun facts - what does your team eat for breakfast?
 🧙 Remember, you can do mighty things with the power of [Markdown](https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)
 -->
+
+As “Big Data” cancer researchers, we both apply existing techniques and continuously work to improve them and create new methods to create clinically useful diagnostics that personalize therapy.
```

Comment:
Two `create` in the same sentence. I like the original sentence from the website:
```
As “Big Data” researchers, we both apply existing techniques and continuously work to improve them and create new methods. We hope to influence and facilitate the next generation of researchers, who will use these techniques to cure cancer and improve human health and well-being in ways we can’t imagine today!
```

Reply:
Updated. 

I moved the second sentence you referenced to include a "call to action" so people will feel more compelled to use our tools so we can get more feedback. What do you think? Should I leave it as the original, too? And just write the request for feedback more simply as a separate thing?

Reply:
I think the current version works~